therapy/etl/drugsatfda.py

```diff
@@ -0,0 +1,157 @@
+"""ETL methods for the Drugs@FDA source."""
+from typing import List, Optional
+import zipfile
+from io import BytesIO
+import json
+import shutil
+
+import requests
+
+from therapy import DownloadException, logger
+from therapy.schemas import SourceMeta, SourceName, NamespacePrefix, ApprovalStatus, \
+    RecordParams
+from therapy.etl.base import Base
+
+
+class DrugsAtFDA(Base):
+    """Core Drugs@FDA import class."""
+
+    def _download_data(self) -> None:
```

Comment:
This doesn't download latest data file. Unsure if this is fixed in #204 , if not should add drugsatfda to the list

Reply:
👍 I'm doing some extra overhaul in 204 so decided to put it off until then